packages/page-staking/src/useSortedTargets.ts

```diff
@@ -244,6 +247,28 @@ function extractBaseInfo (api: ApiPromise, allAccounts: string[], electedDerive:
   };
 }
 
+const b = (x: BN, api: ApiPromise): string => api.createType('Balance', x).toHuman();
```

Comment:
I think there's already a `formatBalance` or something in the `/api/utils` that might be more ergonomic to use in this repo.